models/pd/prompts_pd.py

```diff
@@ -65,6 +66,26 @@ class ExampleUpdateModel(ExampleModel):
     id: int
 
 
+class VariableModel(BaseModel):
+    prompt_id: int
+    name: str
```

Comment:
Just a tip:
you can use this structure to make a regex field:
name: constr(regex=r'^[a-zA-Z_][a-zA-Z0-9_]*$')